src/Polly.Core/Hedging/HedgingActionGeneratorArguments.cs

```diff
@@ -4,10 +4,15 @@ namespace Polly.Hedging;
 /// Represents arguments used in the hedging resilience strategy.
 /// </summary>
 /// <typeparam name="TResult">The type of the result.</typeparam>
-/// <param name="Context">The context associated with the execution of a user-provided callback.</param>
+/// <param name="PrimaryContext">The primary resilience context.</param>
+/// <param name="Context">The context associated with the execution of a user-provided callback that is cloned from the primary context.</param>
```

Comment:
Could we prefix this to give it a complimentary name to `PrimaryContext`? (`SecondaryContext`?)